src/rules/monitors.ts

```diff
@@ -456,6 +456,132 @@ graphite: {}
   },
 };
 
+export const carbonPlaintextPortValid: YAMLRule = {
+  name: "prop-carbon-plaintext-port-valid",
+  type: "error",
+  message: "If specified, `carbon.plaintext_port` must be a valid TCP port",
+  test: {
+    Dot: {
+      path: "carbon",
+      pred: {
+        And: {
+          preds: [
+            { Truthy: { path: "plaintext_port" } },
+            {
+              Or: {
+                preds: [
+                  { GT: { path: "plaintext_port", value: 65535 } },
```

Comment:
Is this right, "greater than 65535 and less than 0"?

Reply:
its copy pasted